parachains/runtimes/bridge-hubs/README.md

```diff
@@ -234,22 +236,57 @@ RUST_LOG=runtime=trace,rpc=trace,bridge=trace \
 	- Wockmint (see `xcmpQueue.Success` for `transfer-asset` and `xcmpQueue.Fail` for `ping-via-bridge`) https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fws-wococo-wockmint-collator-node-0.parity-testnet.parity.io#/explorer
 	- BridgeHubRococo (see `bridgeWococoMessages.MessagesDelivered`)
 
-## How to test local BridgeHubKusama
+## How to test locally Kusama <-> Polkadot bridge
+
+Check [requirements](#requirements-for-local-runtesting) for "sudo pallet + fast-runtime".
+
+### 1. Run chains (Kusama + BridgeHub, Polkadot + BridgeHub) with zombienet
+
+```
+# Kusama + BridgeHubKusama + Statemine
+POLKADOT_BINARY_PATH=~/local_bridge_testing/bin/polkadot \
+POLKADOT_PARACHAIN_BINARY_PATH=~/local_bridge_testing/bin/polkadot-parachain \
+POLKADOT_PARACHAIN_BINARY_PATH_FOR_STATEMINE=~/local_bridge_testing/bin/polkadot-parachain-mint \
+	~/local_bridge_testing/bin/zombienet-linux --provider native spawn ./zombienet/bridge-hubs/bridge_hub_kusama_local_network.toml
+```
+
+```
+# TODO:check-parameter - once Statemint is ready, switch it here instead of Westmint
+# Polkadot + BridgeHubPolkadot + Westmint
+POLKADOT_BINARY_PATH=~/local_bridge_testing/bin/polkadot \
+POLKADOT_PARACHAIN_BINARY_PATH=~/local_bridge_testing/bin/polkadot-parachain \
+POLKADOT_PARACHAIN_BINARY_PATH_FOR_STATEMINT=~/local_bridge_testing/bin/polkadot-parachain-mint \
+STATEMINT_CHAIN=westmint-local \
+	~/local_bridge_testing/bin/zombienet-linux --provider native spawn ./zombienet/bridge-hubs/bridge_hub_polkadot_local_network.toml
```

Comment:
IIRC Zombienet needs some updating to `asset-hub-{relay}`, but would be nice to make these consistent with the new naming.